style.css

```diff
@@ -503,3 +513,247 @@ a {
+    background: #fff;
+    height: 732px;
+    display: flex;
+    flex-direction: column;
+    background-image: url('images/headerbgdisktop.svg');
+    background-size: 75%;
+    background-position: center;
+    background-repeat: no-repeat;
+  }
+
+  .navbar {
+    padding-inline-start: 150px;
+    padding-inline-end: 150px;
+  }
+
+  .section {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    width: 100%;
+    height: 100%;
+    align-items: center;
+  }
+
+  .heading {
+    width: 245px;
+    height: 66px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 28px;
+    line-height: 30px;
+  }
+
+  .disktop-heading {
+    display: flex;
+    flex-direction: column;
+    gap: 9px;
+    width: 606px;
+    height: 445px;
+    margin-top: 93px;
+  }
+
+  .description {
+    width: 581px;
+    height: 74px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 13px;
+    line-height: 22px;
+    color: #344563;
+  }
+
+  .btn-connect {
+    width: 125px;
+    height: 35px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+  }
+
+  .card-section .card {
+    width: auto;
+    height: 333px;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .card-body .card-title {
+    width: 496px;
+    height: 49px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 31px;
+    line-height: 67px;
+    color: #091e42;
+  }
+
+  .card-section {
+    width: 100%;
+    height: auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border: 1px solid #dfe1e6;
+    background: #e5e5e5;
+    justify-content: center;
+    padding-bottom: 80px;
+  }
+
+  .order-1 {
+    order: 1;
+  }
+
+  .order-0 {
+    order: 0;
+  }
+
+  .card-section .card .card-heading img {
+    display: block;
+    width: 295px;
+    height: 326px;
+    margin-top: 0;
+    border-radius: 8px;
+  }
+
+  .card-description {
+    width: 370px;
+    height: 50px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .btn-card-one {
+    width: 100px;
+    height: 24px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    color: #396df2;
+    margin-top: 30px;
+  }
+
+  .ab-section {
+    width: 70%;
+    height: 333px;
+    display: flex;
+  }
+
+  .about-section h1 {
+    margin-top: 0;
+    width: 178px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 29px;
+    line-height: 44px;
+    color: #172b4d;
+  }
+
+  .about-section {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    gap: 20px;
+  }
+
```

Comment:
- [ ] As I mentioned above in the first comment, I think what **I mentioned will help you implement the top-right rounded** corner. As I said **this really enhances the User interface and makes your portfolio unique** 😉.  Please use the same technique to implement the curve on the about section.

**Figma design**
![image](https://user-images.githubusercontent.com/90222110/186998877-051fafee-4c0d-4591-80e1-5d144fc8f72f.png)

**Your design**
![image](https://user-images.githubusercontent.com/90222110/186998922-ff48a67f-ca37-4aee-b409-b3faf0086b8f.png)

**NOTE** Remember to set the background color of body to grey (i.e: #E5E5E5)